src/hooks/useCroct.test.tsx

```diff
@@ -1,28 +1,31 @@
-import {renderHook} from '@testing-library/react-hooks';
+import {renderHook} from '@testing-library/react';
 import croct, {Plug} from '@croct/plug';
 import {useCroct} from './useCroct';
 import {CroctContext, CroctProviderProps} from '../CroctProvider';
 
 describe('useCroct', () => {
     it('should fail if used out of the <CroctProvider/> component', () => {
-        jest.spyOn(console, 'error').mockImplementation();
+        jest.spyOn(console, 'error')
+            .mockImplementation();
 
-        const {result} = renderHook<CroctProviderProps, Plug>(() => useCroct());
+        expect(() => {
+            renderHook<Plug, CroctProviderProps>(() => useCroct());
+        })
+            .toThrow('useCroct() can only be used in the context of a <CroctProvider> component.');
 
         // eslint-disable-next-line no-console
-        expect(console.error).toHaveBeenCalled();
-
-        expect(result.error?.message)
-            .toBe('useCroct() can only be used in the context of a <CroctProvider> component.');
+        expect(console.error)
+            .toHaveBeenCalled();
     });
 
     it('should return the Plug instance', () => {
-        const {result} = renderHook<CroctProviderProps, Plug>(() => useCroct(), {
+        const {result} = renderHook<Plug, CroctProviderProps>(() => useCroct(), {
             wrapper: ({children}) => (
                 <CroctContext.Provider value={{plug: croct}}>{children}</CroctContext.Provider>
             ),
         });
```

Comment:
We can improve the indenting and change the first generic type to `PropsWithChildren` since `children` is the only prop our wrapper receives.

```suggestion
        const {result} = renderHook<PropsWithChildren, Plug>(
            () => useCroct(),
            {
                wrapper: ({children}) => (
                    <CroctContext.Provider value={{plug: croct}}>
                        {children}
                    </CroctContext.Provider>
                ),
            },
        );
```